This is to fix vulnerability 88760f4d-8ef7-11ea-a66d-4b2ef158be83,
and was provided by Mark Sapiro, the upstream maintainer.

Source: https://bugs.launchpad.net/mailman/+bug/1873722

--- Mailman/Cgi/options.py	2019-03-06 17:48:32 +0000
+++ Mailman/Cgi/options.py	2020-04-20 03:10:16 +0000
@@ -173,7 +173,7 @@
     try:
         Utils.ValidateEmail(user)
     except Errors.EmailAddressError:
-        doc.addError(_('Illegal Email Address: %(safeuser)s'))
+        doc.addError(_('Illegal Email Address'))
         loginpage(mlist, doc, None, language)
         print doc.Format()
         return

